{{cookiecutter.project_slug}}/export.sh

```diff
@@ -0,0 +1 @@
+env >> .env
```

Comment:
I don't think this needs to be a file. Instead it would be cleaner to put it inline in `config.yml`.

Reply:
Agreed